### PR TITLE
MGMT-3507 Openshift version is sent to relevant metrics

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

File: internal/metrics/metricsManager.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func NewMetricsManager(registry prometheus.Registerer) *MetricsManager {
 			Subsystem: subsystem,
 			Name:      counterClusterHostInstallationCount,
 			Help:      counterDescriptionClusterHostInstallationCount,
-		}, []string{clusterIdLabel, emailDomainLabel}),
+		}, []string{openshiftVersionLabel, clusterIdLabel, emailDomainLabel}),
```
Comment (nmagnezi) [inline internal/metrics/metricsManager.go:118]: you should provide the openshift version in [line#209](https://github.com/openshift/assisted-service/pull/870/files#diff-6d51a205c8e038996615c6ed4baa7cb8822fe75ef259ad398f66be80097f8600R209)

File: internal/metrics/metricsManager.go [hunk 2]
```diff
@@ -132,7 +132,7 @@ func NewMetricsManager(registry prometheus.Registerer) *MetricsManager {
 			Help:      counterDescriptionOperationDurationMiliSeconds,
 			Buckets: []float64{100, 200, 300, 400, 500, 600, 700, 800, 900, 1000, 1200,
 				1400, 1600, 1800, 2000, 2400, 2800, 3200, 3600, 4000, 5000},
-		}, []string{operation}),
+		}, []string{openshiftVersionLabel, operation}),
```
Comment (nmagnezi) [inline internal/metrics/metricsManager.go:135]: you should provide the openshift version in [line#215](https://github.com/openshift/assisted-service/pull/870/files#diff-6d51a205c8e038996615c6ed4baa7cb8822fe75ef259ad398f66be80097f8600R215)

**Review by nmagnezi — COMMENTED** [review]
You should provide the openshift version info to the label.
Additionally, you'll need to `skipper make generate` to update `mock_metricManager`

**Review by nmagnezi — COMMENTED** [review]
/test assisted-service-aws

**Review by ronniel1 — COMMENTED** [review]
@nmagnezi , @ErezAlster is `ClusterHostInstallationCount` the only metric that needs to be changed?

**Comment by ErezAlster** [comment]
> @nmagnezi , @ErezAlster is `ClusterHostInstallationCount` the only metric that needs to be changed?

yes, all the others has already the openshift label.

**Pull request merged**